src/com/oltpbenchmark/benchmarks/tpch/TPCHLoader.java

```diff
@@ -282,18 +283,28 @@ protected long loadHelper() {
         loaders[6] = loadRegions();
         loaders[7] = loadSuppliers();
 
-        for (int i = 0; i < 8; ++i)
-            if (loaders[i] != null)
-                loaders[i].start();
-
+//        for (int i = 0; i < 8; ++i)
```

Comment:
What did you change here?

Reply:
This part changes the loading from multiple threads at the same time to one at a time. The reason is some unknown issue (till now) of comdb2 (or something between oltp and comdb2) part that stopped multi-thread running. Should we keep some comment here or just revert everything?